docs/quickstart.md

```diff
@@ -0,0 +1,760 @@
+                    Type: Api 
+                    Properties:
+                        Path: /hello/{name}
+                        Method: get
+    Outputs:
+        HelloWorldApi:
+            Description: "API Gateway endpoint URL for Prod stage for Hello World function"
+            Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/hello/"
+    ```
+
+Few things happened here. Firstly, we imported required x-ray SDK classes. 
+`xray_recorder` is instance of global AWS X-Ray recorder class that will begin/end segments/subsegments and send them to the X-Ray daemon.
+In order to create new sub-segments we use `xray_recorder.in_subsegment` method as context manager. 
+By using custom subsegments we were able to add granular sub-traces visible in our X-RAY console, separated by called method name.
+Also we track lambda cold start by setting global variable outside of handler. Variable is set only during lambda runtime initialization. 
+In order to enable tracing for our Lambda we need to configure it in our SAM template.yaml and add `Tracing: Active` under lambda `Properties` section.
+!!! Info 
+    Interested in diving deep in Context Managers and understanding benefits of using them? Follow [article](https://realpython.com/python-with-statement/) from Real Python.
+!!! Info
+   If you want to understand how Lambda execution environment works and why cold starts may occur, follow great [blog series](https://aws.amazon.com/blogs/compute/operating-lambda-performance-optimization-part-1/.
+
+Can we simplify it, so we don't need to worry about adding segment or subsegments? Lets see how Powertools Lambda can help us here.
+
+=== "app.py"
+
+    ```python hl_lines="1 11 13 19 21 27"
+    from aws_lambda_powertools import Logger, Tracer
+    from aws_lambda_powertools.event_handler.api_gateway import ApiGatewayResolver
+    from aws_lambda_powertools.logging import correlation_paths
+
+    logger = Logger(service="APP")
+    tracer = Tracer()
+    app = ApiGatewayResolver()
+
+
+    @app.get("/hello/<name>")
+    @tracer.capture_method
+    def hello_name(name):
+        tracer.put_annotation("User", name)
+        logger.info(f"Request from {name} received")
+        return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+    @app.get("/hello")
+    @tracer.capture_method
+    def hello():
+        tracer.put_annotation("User", "unknown")
+        logger.info(f"Request from unknown received")
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    @logger.inject_lambda_context(correlation_id_path=correlation_paths.API_GATEWAY_REST, log_event=True)
+    @tracer.capture_lambda_handler
+    def lambda_handler(event, context):
+        return app.resolve(event, context)
+    ```
+    
+Thanks to using powertools tracer we have much cleaner code right now. 
+In order to make our methods visible in traces we added `@tracer.capture_method` decorator to processing methods. 
```

Comment:
~~In order~~

Reply:
Rewrote to:
`Thanks to the help of powertools tracer we have much cleaner code right now. 
To make our methods visible in the traces, we added @tracer. capture_method' decorator to the processing methods.`